amundsen_application/static/js/components/common/EditableText/index.tsx

```diff
@@ -1,10 +1,15 @@
-import autosize from 'autosize';
+import * as autosize from 'autosize';
 import * as React from 'react';
-import { Overlay, Tooltip } from 'react-bootstrap';
 import * as ReactMarkdown from 'react-markdown';
 
 // TODO: Use css-modules instead of 'import'
 import './styles.scss';
+import {
+  CANCEL_BUTTON,
```

Comment:
nit nit: `SOMETHING_BUTTON_TEXT` as the name? Without tracing back to the files to see this is a string from constants, the name might confuse someone looking into thinking we were importing some element